config.json

```
@@ -35,6 +35,17 @@
   },
   "exercises": {
     "concept": [
+      {
+        "slug": "annalyns-infiltration",
+        "name": "Annalyn's Infiltration",
+        "uuid": "897fa2ea-c29c-4d4e-9469-ff661a9b838a",
+        "difficulty": 1,
+        "concepts": [
+          "booleans"
+        ],
+        "prerequisites": [],
+        "status": "active"
```

Maybe start out with `wip` to ensure that only track maintainers will see it?

```suggestion
        "status": "wip"
```